pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright © 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>22.1.0</version>
+    </parent>
+
+    <groupId>io.gravitee.policy</groupId>
+    <artifactId>gravitee-policy-transform-status-code</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <name>Gravitee.io APIM - Plugin - Entrypoint HTTP Post</name>
+    <description>Send an HTTP POST request including URL parameters in the request and process the response from the HTTP server.</description>
+
+    <properties>
+        <gravitee-bom.version>8.0.4</gravitee-bom.version>
+        <gravitee-common.version>4.4.0</gravitee-common.version>
+        <gravitee-gateway-api.version>3.5.0</gravitee-gateway-api.version>
+        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-apim.version>4.2.5</gravitee-apim.version>
+        <gravitee-reactor-message.version>1.0.1</gravitee-reactor-message.version>
```

Comment:
May changed if the policy has to be compatible with 4.2+ or 4.6+.
This comment is just a reminder, to avoid merging the PR by mistake

Reply:
Yes, good point.